src/psyclone/tests/psyir/transformations/reference2arrayrange_trans_test.py

```diff
@@ -218,6 +217,26 @@ def test_validate_range(fortran_reader):
            "'ArrayReference'." in str(info.value))
 
 
+def test_validate_structure(fortran_reader):
+    '''Test that a StructureReference raises an exception.'''
```

Comment:
Please say that this is a limitation that will be tackled by #1858.

Reply:
Added.